book/accessibility.md

```diff
@@ -5,23 +5,80 @@ prev: animations
 next: embeds
 ...
 
-It's important for *everyone* to be able to access content on the web, even if
-you have a hard time reading small text, can't or don't want to use a mouse,
-are triggered by very bright colors, suffer from repetitive stress injury, or
-can't see a computer screen at all. Browsers have
-[*accessibility*][a11y] features aimed at all of these use cases, taking
-advantage of the fact that web pages [declare] UI and allow the browser to
-manipulate it [on behalf][ua] of the user.  On top of these features, browsers
-need to provide a robust set of features to help web sites avoid losing
-accessibility when customizing their look and feel. Accessiblity is a critical
-part of a modern browser.
-
-[ua]: http://localhost:8001/intro.html#the-role-of-the-browser
+So far, we've focused on making the browser an effective platform for
+developing web applications. But ultimately, the browser is a [_user_
+agent][ua]. That means it should assist the user in whatever way it
+can to access and use web applications. Browsers therefore offer a
+range of [*accessibility*][a11y] features that take advantage of
+[declarative] UI and the flexibility of HTML and CSS to make it
+possible to interact with the web page by touch, keyboard, or voice.
+
+[ua]: intro.md#the-role-of-the-browser
 
 [declare]: intro.md#browser-code-concepts
 
 [a11y]: https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility
 
+What is accessibility
+=====================
+
+"Accessibility" refers to any way a user can change or customize how
+they interact with a web page,[^not-just-screen-reader] and the web's
```

Comment:
```suggestion
they interact with a web page in order to make it to use,[^not-just-screen-reader] and the web's
```

Reply:
Add another footnote acknowledging other definitions, such as "the practice of making your websites usable by as many people as possible" [1] (which is a framing of the same thing from a web developer point of view

[1] https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility